### PR TITLE
feat: iOS 26 glass

### PR DESCRIPTION
**Pull request by ncoelho**
Minimal iOS 26 implementation. Only headers using react navigation. And on Home view + Wallet view. Other views need (minimal) work on the headers, but should be done on different PRs.

https://github.com/user-attachments/assets/63fdd215-a38e-400b-948f-57e5f3750827

### CODE REVIEW

**Review by GladosBlueWallet — COMMENTED** [review]
Review request for BlueWallet/BlueWallet#8508: feat: iOS 26 glass

Requested checks before merge:
1) Security/secret handling (no sensitive logging, validate inputs).
2) Edge-case correctness + error handling (fail closed).
3) Tests/coverage for critical paths and regressions.
4) Risk assessment and any required follow-ups.

Reason: open_unreviewed_pr

File: navigation/DetailViewScreensStack.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React, { lazy, useCallback, useMemo } from 'react';
 import { View, Platform, PlatformColor } from 'react-native';
-import { NativeStackNavigationOptions } from '@react-navigation/native-stack';
+import { NativeStackNavigationOptions, NativeStackHeaderItem } from '@react-navigation/native-stack';
```
Comment (GladosBlueWallet) [inline navigation/DetailViewScreensStack.tsx]: `NativeStackHeaderItem` may not be a public export in @react-navigation/native-stack (version-dependent). This can turn into a hard TS build break. Safer: avoid the named import and type the items as `NonNullable<NativeStackNavigationOptions['unstable_headerRightItems']> extends (...args:any)=>infer R ? R : never`, or use the officially exported type (often `NativeStackHeaderRightItem`/similar) for your installed version.

File: navigation/DetailViewScreensStack.tsx [hunk 2]
```diff
@@ -83,6 +96,46 @@ const DetailViewStackScreensStack = () => {
...
+      // action with a long-press menu.
+      return {
+        title: sizeClass === SizeClass.Large ? loc.wallets.list_title : '',
+        headerLargeTitle: false,
+        headerTransparent: true,
+        unstable_headerRightItems: () => {
+          if (isDesktop) {
+            return [];
+          }
+          const items: NativeStackHeaderItem[] = [
```
Comment (GladosBlueWallet) [inline navigation/DetailViewScreensStack.tsx]: This explicit `NativeStackHeaderItem[]` annotation is the point where a missing/renamed type will explode. Consider letting TS infer from `unstable_headerRightItems` return type, or alias the type from `NativeStackNavigationOptions` so it tracks upstream changes automatically.

**Review by GladosBlueWallet — COMMENTED** [review]
Neat glass effect. Now for the boring part: compile safety.

Primary concern: type imports for `unstable_headerRightItems` look non-portable across @react-navigation/native-stack versions.

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Review by Overtorment — APPROVED** [review]
lgtm
